src/normalize/core.ts

```diff
@@ -65,6 +65,7 @@ export class CoreNormalizer extends SpecMapper<NormalizerParams, FacetedUnitSpec
 
     const specWithReplacedEncoding = {
       ...spec,
+      name: spec.name && params.repeaterPrefix ? params.repeaterPrefix + spec.name : undefined,
```

Comment:
This will remove the name when there is no repeater prefix, which is definitely wrong (see changed example in this diff).